avocado/plugins/assets.py

```diff
@@ -0,0 +1,179 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2019
+# Authors: Willian Rampazzo <willianr@redhat.com>
+
+import ast
+import errno
```

Comment:
This is why the CI is failing (unused import)

Reply:
... btw please use `pylint` and fix as much issues as possible (some doesn't make sense, but in most cases we do want 10 out of 10 score...)

Reply:
... also, cool PR number and about the commit message, it's not necessary to include the `This is the v1 ...` stuff. Simply describe what this is trying to solve and eventually the limitations....

Reply:
> This is why the CI is failing (unused import)

This was forgotten as part of a previous implementation idea. I already removed it from the next version.

> ... btw please use `pylint` and fix as much issues as possible (some doesn't make sense, but in most cases we do want 10 out of 10 score...)

Sure, will do.

> ... also, cool PR number and about the commit message, it's not necessary to include the `This is the v1 ...` stuff. Simply describe what this is trying to solve and eventually the limitations....

I just wanted to make clear that this implementation is not intended to be a final version. I will improve it in the next version.

Reply:
> ... btw please use `pylint` and fix as much issues as possible (some doesn't make sense, but in most cases we do want 10 out of 10 score...)

I just noticed pylint was disabled in my editor and I thought it was enabled. I enabled it and now I see all the pylint warning. Thanks for that!